templates/invoices/index.html

```diff
@@ -534,30 +542,29 @@ <h6 class="text-subtitle1 q-my-none">
             LNbits.utils.notifyApiError(error)
           })
       },
-      deleteTPoS: function (tposId) {
-        var self = this
-        var tpos = _.findWhere(this.tposs, {id: tposId})
+      deleteInvoice(invoice_id) {
+        const adminkey = this.g.user.wallets[0].adminkey
+        // console.log("admin key ....", adminkey)
 
         LNbits.utils
-          .confirmDialog('Are you sure you want to delete this TPoS?')
-          .onOk(function () {
+        .confirmDialog('Are you sure you want to delete this Invoice?')
+        .onOk(() => {
+            // console.log("Deleting invoice...", invoice_id)
+
             LNbits.api
-              .request(
-                'DELETE',
-                '/tpos/api/v1/tposs/' + tposId,
-                _.findWhere(self.g.user.wallets, {id: tpos.wallet}).adminkey
-              )
-              .then(function (response) {
-                self.tposs = _.reject(self.tposs, function (obj) {
-                  return obj.id == tposId
-                })
-              })
-              .catch(function (error) {
-                LNbits.utils.notifyApiError(error)
-              })
-          })
+            .request(
+              'DELETE',
+              '/invoices/api/v1/invoice/' +  invoice_id + '/delete', adminkey
+              ).then(response => { 
+                if (response.status == 200) {
+                  this.invoices = _.reject(this.invoices, obj => obj.id === invoice_id)
+                }
+            }).catch(err => { 
+              LNbits.utils.notifyApiError(err)
+            })
+        })
       },
-      exportCSV: function () {
+     exportCSV: function () {
```

Comment:
1 white space missing, maybe you can run prettier?

Reply:
ok ran prettier on whole file